.github/workflows/main.yml

```diff
@@ -10,9 +10,70 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-name: Lint
+name: Pull request
 on: [pull_request, merge_group]
 jobs:
+  decide-jobs:
+    name: Decide which jobs to run
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'Qiskit' && github.event_name == 'pull_request'
```

Comment:
`github.event_name == 'pull_request'` prevents the notebook tester and api checks from running in the merge queue. This is the same behaviour as before.